multicluster/controllers/multicluster/resourceexport_controller.go

```diff
@@ -203,6 +205,7 @@ func (r *ResourceExportReconciler) handleDeleteEvent(ctx context.Context, resExp
 		return err
 	}
 	resImportName := GetResourceImportName(resExport)
+	klog.Infof("There is resImport to delete named %s", resImportName)
```

Comment:
s/resImport/ResourceImport/

Reply:
Thanks. This was a testing log. I'll re-format it to a proper one.